bigbluebutton-html5/imports/api/users/server/modifiers/removeUser.js

```diff
@@ -43,7 +43,12 @@ export default function removeUser(meetingId, userId) {
 
     clearUserInfoForRequester(meetingId, userId);
 
-    Users.remove(selector);
+    Meteor.wrapAsync((callback) => {
+      Meteor.setTimeout(() => {
```

Comment:
Please add inline comment describing why you add timeout

What happens if the user was on bad connection and the 1 second is not enough to get the update - i.e. gets update about user removed and user ejected at the same time - any errors, any odd behaviour?

Reply:
> Please add inline comment describing why you add timeout

Comment added in 98ad928

> What happens if the user was on bad connection and the 1 second is not enough to get the update - i.e. gets update about user removed and user ejected at the same time - any errors, any odd behaviour?

If user removed happens before user ejected the user will see `ErrorScreen` instead of `MeetingEnded` (same behavior we had without the PR). That's why this workaround should be removed as soon as we have a proper fix to that issue